authentication.go

```diff
@@ -28,7 +27,9 @@ func setSignature(header *http.Header, xPriv *bip32.ExtendedKey, bodyString stri
 	// Set the auth header
 	header.Set(models.AuthHeader, authData.XPub)
 
-	return setSignatureHeaders(header, authData)
+	setSignatureHeaders(header, authData)
```

Comment:
hm i think previous way was better since u not checking here for return
```go
func setSignatureHeaders(header *http.Header, authData *models.AuthPayload) ResponseError {
	// Create the auth header hash
	header.Set(models.AuthHeaderHash, authData.AuthHash)

	// Set the nonce
	header.Set(models.AuthHeaderNonce, authData.AuthNonce)

	// Set the time
	header.Set(models.AuthHeaderTime, fmt.Sprintf("%d", authData.AuthTime))

	// Set the signature
	header.Set(models.AuthSignature, authData.Signature)

	return nil
}
```

Reply:
ah ok i see u changed this `func setSignatureHeaders(header *http.Header, authData *models.AuthPayload) {`

Reply:
so then we good :)